tox.ini

```diff
@@ -56,7 +56,6 @@ commands =
 description = type check ourselves
 commands =
     python -m mypy --config-file mypy_self_check.ini -p mypy -p mypyc
-    python -m mypy --config-file mypy_self_check.ini scripts
```

Comment:
Did you mean to do this as part of this PR? Looks like it maybe should have been done as part of #13286? :)